indico/modules/announcement/templates/display.html

```diff
@@ -1,23 +1,80 @@
-<div class="announcement-bar weak-hidden" id="announcement-bar">
-    <div class="flexrow f-self-stretch f-j-center f-a-center">
-        <i class="icon icon-warning"></i>
-        <div>{{- message | safe -}}</div>
-    </div>
-    <span class="icon-close close-button"></span>
-</div>
-<script>
-    (function() {
-        'use strict';
+{% macro render_announcement_legacy(type, message, persistent=false) %}
+    {% if type == 'html' %}
+        {{ message | safe }}
+    {% else %}
+        <div class="announcement-bar weak-hidden {{type}}" data-hash={{message | crc32 | string | tojson}}>
```

Comment:
Do we really need the `|string|tojson` part?

Reply:
Not if we want to just add the quotes around.

Reply:
using a jsonifier to create the surrounding quotes for an html attribute sounds super dirty :P